pay-api/migrations/versions/6ba89cd39cc8_adding_error_codes_table.py

```diff
@@ -0,0 +1,137 @@
+"""adding_error_codes_table
+
+Revision ID: 6ba89cd39cc8
+Revises: 8eac44955fb6
+Create Date: 2020-04-14 12:36:57.094459
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '6ba89cd39cc8'
+down_revision = '8eac44955fb6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    error_code_table = op.create_table('error_code',
+                                       sa.Column('code', sa.String(length=50), nullable=False),
```

Comment:
the only minor thing i can think is to have some kind of system identifier ;either as a column or as prefix .something  PAY_INVALID_IDENITIFER.
When we move to the common system for errors , there will be huge number of errors

Reply:
Hmm.. I think when we move to common table, there would be an indicator for the system (pay, auth, legal etc.). Do we need to incorporate this now, as this is in our schema ?

Reply:
yeah.. we might not need now.